lib/YuiRestClient.pm

```diff
@@ -78,6 +78,7 @@ sub set_timeout {
 sub init_app {
     my (%args) = @_;
     my $timeout = $args{timeout} || TIMEOUT;
+    $timeout *= get_var('TIMEOUT_SCALE', 1);
```

Comment:
the timeout scale is applicable to specific cases, we don't need for now a general one at the level of the app, otherwise we would bump timeout in too many places, hiding bugs.

Reply:
Done